verticapy/learn/tsa/__init__.py

```diff
@@ -44,3 +44,1801 @@
 #
 # VerticaPy aims to solve all of these problems. The idea is simple: instead
 # of moving data around for processing, VerticaPy brings the logic to the data.
+#
+#
+# Modules
+#
+# Standard Python Modules
+import math, warnings
+
+# VerticaPy Modules
+from verticapy.learn.vmodel import *
+from verticapy.learn.linear_model import LinearRegression
+from verticapy import vDataFrame
+from verticapy.plot import gen_colors
+from verticapy.learn.tools import *
+
+# Other Python Modules
+from dateutil.parser import parse
+import matplotlib.pyplot as plt
+
+# ---#
```

Comment:
Why have you moved the implementation of classes and functions into the __init__.py file?
Afaik, the primary use of __init__.py is to initialize Python packages and not to hold content.

Reply:
I think it would be more appropriate if you move them into a base.py file in this directory, and import it here if needed.

Reply:
Because I'll move it to tsa.py just right after when I will erase the deprecated message which is in the files models and tools.